config.yaml

```diff
@@ -20,38 +20,77 @@ stale_assigned:
     stale_days: 14
     warning_days: 7
     warning_label: "stale-assigned"
-    warning_comment: 'This issue is assigned but has not received an update in {stale_days} days so it has been labeled "{warning_label}". If you are still working on the issue or are waiting for feedback from a Committer, please remove the label and optionally provide an update. If you are waiting for feedback, please consider this a reminder *to Committers/Reviewer* to get back to you. If you are no longer working on the issue, please unassign yourself so someone else may work on it. In {warning_days} days the issue will be automatically unassigned.'
     done_label: "auto-unassigned"
-    done_comment: 'This issue was marked "{warning_label}" and has not received an update in {warning_days} days. It is now automatically unassigned. If you are still working on it, you can assign it to yourself again. Please also give an update about the status of the work.'
+    warning_comment: |
+        Hi!
+
```

Comment:
Have you tested that line breaks are forwarded to the comment in Jira as expected?